From f66815e60c27808a3e6738ae91ae0878f7df893d Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Wed, 22 Mar 2023 17:43:57 +0100
Subject: [PATCH 0264/1695] dt-bindings: add RISAB to default bindings config
 for stm32mp25

The RISAB header is now part of default bindings header file for
stm32mp25.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 7ff3db812631d81c06697f0b765472182edde241 dt-bindings: add RISAB to default bindings config for stm32mp25]
Change-Id: I05254c6e12a3e20baf509c2dd9c4612464e2a1e7
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
---
 core/include/drivers/stm32mp_dt_bindings.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/include/drivers/stm32mp_dt_bindings.h b/core/include/drivers/stm32mp_dt_bindings.h
index bbff1d8ff..f3ebb706a 100644
--- a/core/include/drivers/stm32mp_dt_bindings.h
+++ b/core/include/drivers/stm32mp_dt_bindings.h
@@ -29,6 +29,7 @@
 #include <dt-bindings/reset/stm32mp25-resets.h>
 #include <dt-bindings/soc/stm32mp25-rif.h>
 #include <dt-bindings/soc/stm32mp25-rifsc.h>
+#include <dt-bindings/soc/stm32mp25-risab.h>
 #include <dt-bindings/soc/stm32mp25-risaf.h>
 #endif
 
-- 
2.47.2

